app/controllers/coworking_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class CoworkingController < PrisonsApplicationController
+  def new
+    @prisoner = offender(nomis_offender_id_from_url)
+
+    poms = PrisonOffenderManagerService.get_poms(prison_id_from_url)
+    @active_poms, @unavailable_poms = poms.partition { |pom|
+      %w[active unavailable].include? pom.status
+    }
+
+    @current_pom = current_pom_for(nomis_offender_id_from_url)
+    @prison_poms = @active_poms.select{ |pom| pom.position.include?('PRO') }
+    @probation_poms = @active_poms.select{ |pom| pom.position.include?('PO') }
+  end
+
+  def confirm
+    @prisoner = offender(nomis_offender_id_from_url)
+    @primary_pom = PrisonOffenderManagerService.get_pom(
+      prison_id_from_url, primary_pom_id_from_url
+    )
+    @secondary_pom = PrisonOffenderManagerService.get_pom(
+      prison_id_from_url, nomis_staff_id_from_url
+    )
+
+    @event = :allocate_secondary_pom
```

Comment:
Might it be better to use the constants in AllocationVersion for these?

Reply:
@rossjones We can't actually use the constants in AllocationVersion as we need to pass the event hash e.g event[:allocate_primary_pom]. Seems like AllocationVersion then converts that hash into an integer value. The reason the tests currently pass is because this PR does not include the creation of the co-worker so we are not saving an AllocationVersion. I will had another commit to revert the changes made.